### DISCUSSION TITLE
Also having issues with nodes not being set up (Ubuntu)

### DISCUSSION THREAD

**Discussion by dbmanifest** [MRColorR/money4band#47] [Q&A] [answered]
I followed the instructions to the T, containers are running (tho there was an error in a bash script for missing the "-" in "docker-compose" commands. 

But nothing is registering in any node.

**Comment by MRColorR** [comment]
Hello, thank youbfor reaching out.
that's strange please provide some apps logs so i can check as it's working
fine on my two linux testing machines and i can replicate.
Also notice that many months ago the new docker compose has been released
and therese no need to use the old docker-compose,  this could cause issues
as docker will use the the old compose if you add - .

Il Sab 20 Mag 2023, 23:25 dbmanifest ***@***.***> ha scritto:

**Comment by dbmanifest** [comment, marked as answer]
I was able to get it working, I had to reset both the environment file and
the docker-compose yaml and it rebuilt everything just fine.

About docker compose, I must have an out of date version, which is weird
cause I routinely do updates. Docker compose -d threw an error but
docker-compose -d did the trick.

**Comment by MRColorR** [comment]
Maybe, anyway I'll run another tests routine this on Wednesday when I'll
release the new version with the updated Windows script. So I'll look into
the Linux version again as well, just to be sure :)

Have a nice Sunday to all of you and thank you for using my free opensource
project

Il Dom 21 Mag 2023, 08:18 dbmanifest ***@***.***> ha scritto:

> I was able to get it working, I had to reset both the environment file and
> the docker-compose yaml and it rebuilt everything just fine.
>
> About docker compose, I must have an out of date version, which is weird
> cause I routinely do updates. Docker compose -d threw an error but
> docker-compose -d did the trick.
>

**Comment by dbmanifest** [comment]
Awesome, thank you. It could have just been me being a moron, I was
multitasking as I was spinning it up.

Is there a way I could email or private message you? I have some ideas I
want to bounce off of you that I don't necessarily want to be public at the
moment.

**Comment by MRColorR** [comment]
Yes, feel free to reach to me also on: ***@***.***

Il Dom 21 Mag 2023, 08:33 dbmanifest ***@***.***> ha scritto:

> Awesome, thank you. It could have just been me being a moron, I was
> multitasking as I was spinning it up.
>
> Is there a way I could email or private message you? I have some ideas I
> want to bounce off of you that I don't necessarily want to be public at the
> moment.
>
